.github/workflows/publish-release.yml

```diff
@@ -3,7 +3,6 @@ name: Publish
 on:
   release:
     types: [published]
-  workflow_dispatch:
```

Comment:
Just curious: what is the reason to remove this? Don't you think it might be useful to keep it?

Reply:
I added it on the previous attempt thinking we can use this workflow to release on demand manually, but apparently it didn't work, so I remove it.

Reply:
I see, thanks for the clarification